src/Wpf.Ui/Styles/Controls/Button.xaml

```diff
@@ -440,4 +440,92 @@
     <Style BasedOn="{StaticResource DefaultButtonStyle}" TargetType="{x:Type Button}" />
     <Style BasedOn="{StaticResource DefaultUiButtonStyle}" TargetType="{x:Type controls:Button}" />
 
+
+    <!--  TODO I suppose this should not be here  -->
```

Comment:
We should add a back button style for each navigation, either a universal control like `NavigationBackButton` or just drop the style into a separate file.

Reply:
I think it's better to add a separate control with name `NavigationBackButton`.